bin/rechunker

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+import os.path
+
+import strax
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Rechunker for FileSytemBackend. Interfaces with strax.rechunker."
+                    "Please see the documentation of strax.rechunker for more information: "
+                    "github.com/AxFoundation/strax/blob/31c114c5f8329e53289d5127fb2125e71c3d6aae/strax/storage/files.py#L371")
+    parser.add_argument(
+        '--source',
+        type=str,
+        help="Target directory to rechunk, should be a folder in a "
+             "strax.DataDrictory (one datatype)")
+    parser.add_argument(
+        '--dest', '--destination',
+        default=None,
+        dest='dest',
+        type=str,
+        help='Where to store rechunked data. If nothing is specified, replace the source.',
+    )
+    parser.add_argument(
+        '--compressor',
+        choices=list(strax.io.COMPRESSORS.keys()),
+        help="Recompress using one of these compressors. If nothing specified, "
+             "use the same compressor as the source")
+    parser.add_argument(
+        '--rechunk',
+        default=True,
+        choices=[True, False],
+        type=bool,
+        help="rechunk the data")
+    parser.add_argument(
+        '--target_size_mb', '--target-size-mb',
+        dest='target_size_mb',
+        type=int,
+        default=strax.default_chunk_size_mb,
+        help="Target size MB (uncompressed) of the rechunked data")
+    parser.add_argument(
+        '--write_stats_to', '--write-stats-to',
+        dest='write_stats_to',
+        type=str,
+        default=None,
+        help="Write some information to this file (csv format)")
```

Comment:
Maybe add that this requires pandas.

Reply:
Could, but I'm quite sure it's already imported at `import strax` (that being said, might as well add the `import pandas` to the top of the file for good coding practice)

Reply:
okay, just moved it after the argparsing together with import strax